src/pkg/config/token.go

```diff
@@ -121,3 +126,73 @@ func SaveSecretConfig(conf SecretConfig) error {
 
 	return nil
 }
+
+func SaveSelectedOrganization(orgConf OrganizationConfig) error {
```

Comment:
there's a lot of code duplication between this and the other `Save/Load` code. Can you dedup?

Reply:
Also - the name of the file here is `token.go` (should probably be `credentials.go`) and the function handles save/load for the organization file (separate from the token). So it should be in a separate go file.

Reply:
Refactored + renamed to persistent_config